### PR TITLE
remove dependency on jest

### PR DESCRIPTION
**Pull request by psabharwal123**
To fix https://github.com/preactjs/eslint-config-preact/issues/31

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -23,22 +23,19 @@ export default {
 
 	// We don't use plugin:react/recommended here to avoid React-specific rules.
 	extends: [
-		'eslint:recommended',
-		'plugin:jest/recommended'
+		'eslint:recommended'
 	],
 
 	// TODO: preact-cli ships Jest now, Mocha may be a candidate for removal.
```
Comment (rschristian) [inline index.js:29]: Can probably get rid of this

```suggestion
```

File: package.json [hunk 2]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "eslint-config-preact",
-  "version": "1.4.0",
+  "version": "1.5.0",
```
Comment (rschristian) [inline package.json:3]: I'll increment this separately, let's keep it out of the PR

**Review by rschristian — COMMENTED** [review]
Thanks! Couple comments, but I appreciate you taking the time to do this.

We should probably remove the `eslint-plugin-jest` dependency too.

**Comment by psabharwal123** [comment]
np! made the changes

**Comment by psabharwal123** [comment]
@rschristian reminder to publish this :)

**Comment by rschristian** [comment]
Did so yesterday: https://github.com/preactjs/eslint-config-preact/releases/tag/1.5.0

**Pull request merged**